src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/Search.js

```diff
@@ -22,6 +23,24 @@ export default class Search extends React.Component<Props> {
         this.value = value;
     }
 
+    updateState(value: ?string) {
```

Comment:
`updateState` is a very generic name... Can't we come up with something more specific? Maybe `updateValue`?

Reply:
It's not just an update of the value here..